recipes/parallel-hashmap/all/conanfile.py

```diff
@@ -10,6 +10,7 @@
 
 class ParallelHashmapConan(ConanFile):
     name = "parallel-hashmap"
+    version = "1.3.11"
```

Comment:
```suggestion
```

Reply:
I removed it, but when I run `conan create .` locally using conan v2, it fails if the version is not present.